packages/common/container-definitions/src/deltas.ts

```diff
@@ -61,21 +61,94 @@ export interface IDeltaSender {
     flush(): void;
 }
 
-/** Events emitted by the Delta Manager */
+/**
+ * Events emitted by {@link IDeltaManager}.
+ */
+/* eslint-disable @typescript-eslint/unified-signatures */
 export interface IDeltaManagerEvents extends IEvent {
+    /**
+     * TODO
+     *
+     * @remarks Listener parameters:
+     *
+     * - `messageBuffer`: TODO
+     */
     (event: "prepareSend", listener: (messageBuffer: any[]) => void);
+
+    /**
+     * TODO
+     *
+     * @remarks Listener parameters:
+     *
+     * - `message`: TODO
+     */
     (event: "submitOp", listener: (message: IDocumentMessage) => void);
+
+    /**
+     * TODO
+     *
+     * @remarks Listener parameters:
+     *
+     * - `message`: TODO
+     *
+     * - `processingTime`: TODO
+     */
     (event: "op", listener: (message: ISequencedDocumentMessage, processingTime: number) => void);
+
+    /**
+     * TODO
+     */
     (event: "allSentOpsAckd", listener: () => void);
-    (event: "pong" | "processTime", listener: (latency: number) => void);
+
+    /**
+     * TODO
+     *
+     * @remarks Listener parameters:
+     *
+     * - `latency`: TODO
+     */
+    (event: "pong", listener: (latency: number) => void);
+
+    /**
+     * TODO
+     *
+     * @remarks Listener parameters:
+     *
+     * - `latency`: TODO
+     */
+    (event: "processTime", listener: (latency: number) => void);
+
     /**
-     * The connect event fires once we've received the connect_document_success message from the
-     * server.  This happens prior to the client's join message (if there is a join message).
+     * Emitted once we've received the connect_document_success message from the server.
```

Comment:
Is this a reasonable semantic description at this level in the API? Or should this be more general?